tutor/serialize.py

```diff
@@ -17,6 +17,10 @@ def load_all(stream: str) -> t.Iterator[t.Any]:
     return yaml.load_all(stream, Loader=yaml.SafeLoader)
 
 
+def dump_all(content: t.Any, fileobj: TextIOWrapper) -> None:
```

Comment:
I've changed it to `t.Sequence` so that it works with the current `pyYAML` version. 

They recently (2 weeks ago) changed the type to [`t.Iterable`](https://github.com/python/typeshed/commit/8aa9b37a9712a9afc7e3b2e2e45595f5ae3eccc6). Should I bump the YAML version instead?

Reply:
It's fine as it is, as long as tests pass. Those who upgrade pyyaml will take care of upgrading the signature.